pkgs/development/python-modules/scipy/default.nix

```diff
@@ -19,10 +21,15 @@
 , libxcrypt
 }:
 
+assert blas.provider == numpy.blas;
+
 buildPythonPackage rec {
   pname = "scipy";
   version = "1.10.1";
-  format = "pyproject";
+
+  # Required to pass -Dblas options
+  # https://github.com/scipy/scipy/issues/17244
+  format = "build";
```

Comment:
keep it as normal, and instead add your hook in `nativeBuildInputs` and disable the other hook that does the building with an env var.

Reply:
Sounds great, didn't know that was an option